profile/UserProfile.css

```diff
@@ -0,0 +1,52 @@
+.top_bar {
+  width: 100%;
+
+  height: 40px;
+  background-color: #008CC8;
+  border-bottom: 5px solid #008CC8;
+  margin: 0;
+  padding: 0;
+  display: inline-flex;
+	position: absolute;
+	z-index: 10;
+
+}
+
+.blue_box {
+position: absolute;
+width: 1440px;
+left: 0px;
+background: #015579;
+padding: 2em;
+margin: 0;
+}
```

Comment:
Check out the .container utility in main.css. Also, this should probably be a block element.